### DISCUSSION TITLE
Rate limit

### DISCUSSION THREAD

**Discussion by Brainfighter** [crunchy-labs/crunchy-cli#279] [Ideas]
Is it possible to pause the download for x minutes and continue automatically afterwards when the error: You've probably hit a rate limit occurs ?

**Comment by bytedream** [comment]
No it's not. I don't really know how the rate limit works, it might be the case that it gets extended when trying to request the endpoint which is currently rate limited. This is one thought which keeps me away from implementing such a feature
